components/signup/RolesPage.tsx

```diff
@@ -43,7 +43,7 @@ export default ({ student }): JSX.Element => {
           <span style={{ fontWeight: 'bold' }}>Roles</span>
           <span style={{ color: COLORS.GRAY_BB }}> that you would take in your projects</span>
         </Typography>
-        <ArrayForm label="Titles" items={roles} setItems={setRoles} />
+        <ArrayForm label="Titles" items={roles} setItems={setRoles} options={roles} />
```

Comment:
- need to add options prop in line 30, then pass that into ArrayForm's options
- modify pages/signup/setup to fetch from backend (like you did the filter pages)